docs/src/validator_operator/validator_compose.rst

```diff
@@ -185,7 +185,8 @@ AUTH_URL                      The URL of your OIDC provider for obtaining the ``
 AUTH_JWKS_URL                 The URL of your OIDC provider for obtaining the ``jwks.json``, will typically be ``${AUTH_URL}/.well-known/jwks.json``.
 AUTH_WELLKNOWN_URL            The URL of your OIDC provider for obtaining the ``openid-configuration``, will typically be ``${AUTH_URL}/.well-known/openid-configuration``.
 LEDGER_API_AUTH_AUDIENCE      The audience for the participant ledger API. e.g. ``https://ledger_api.example.com``.
-LEDGER_API_AUTH_SCOPE         The scope for the participant ledger API. Optional
+                              This should match the ``ledger-api.auth-services.target-audience`` configuration for the participant, or its documented default if no explicit setting is made.
```

Comment:
I don't understand what you're saying here. This _sets_ the target-audience, so I'm not sure what it means to say that it must match it.